src/api/status/public/js/post/index.js

```diff
@@ -0,0 +1,19 @@
+const totalPostsEl = document.getElementById('totalPosts');
+
+const apiHost = new URL(location).origin;
+
+const fetchPostsCount = () =>
+  fetch(`${apiHost}/v1/posts`, { method: 'HEAD' }).then((res) => res.headers.get('x-total-count'));
```

Comment:
In dev mod, the path `https://dev.api.telescope.cdot.systems/v1/posts` isn't avaiable and the base url is `https://dev.api.telescope.cdot.systems/v1/status`, so we should replace status with `posts`